package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "webgme-executor-worker",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "description": "Worker for connecting to webgme executor framework",
-  "main": "node_worker.js",
+  "main": "index.html",
```

Comment:
Why change this?

Reply:
nw.js reads this field. I reverted my change back to node_worker.js so npm works